.meteor/packages

```diff
@@ -96,3 +96,4 @@ johanbrook:publication-collector
 
 
 # Custom Packages
+natestrauser:select2
```

Comment:
@kieha Also, now that we aren't using this with autoform, can we just use an npm package rather than the Meteor package?